benchmark/ci-helper.js

```diff
@@ -0,0 +1,10 @@
+const benchLogs = process.argv[2];
```

Comment:
Could you add a comment to this file explaining what it is for? It seems to be a particular file used only in CI for doing something that I can't understand 😅

Reply:
Yup! I've written this mostly because I can't write the same thing in bash 😅 I'll note what it's doing